src/plugins/workspace/public/plugin.ts

```diff
@@ -686,9 +652,10 @@ export class WorkspacePlugin
     navigation: NavigationPublicPluginStart
   ) {
     if (contentManagement) {
+      const workspaceClient = coreStart.workspaces.client$.getValue() as WorkspaceClient;
```

Comment:
Is the type cast here needed?

Reply:
I think so because `getValue()` can possibly return null, but since we do initiate `WorkspaceClient` inside `WorkspaceValidationService` so I think the  type assertion is necessary to tell TypeScript that `workspaceClient` can never be null